dts/bindings/ospi/st,stm32-ospi.yaml

```diff
@@ -74,3 +74,65 @@ properties:
     type: boolean
     description: |
       Enables Delay Block (DLYB) Bypass.
+
+  io-low-port:
+    type: string
+    enum:
+      - "IOPORT_NONE"
+      - "IOPORT_1_LOW"
+      - "IOPORT_1_HIGH"
```

Comment:
in which situation would each <io-low-port>  be "IOPORT_n_HIGH" : IOPORT pins are grouped [0:3] and [4:7]
That seems to ease the multiplexing for 2 quad-SPI mode but maybe the enum could be reduced ?

Reply:
Reduced allowed values.